Dockerfile

```diff
@@ -13,7 +13,9 @@ ENV SEGTOOLS_VERSION="1.2.4"
 RUN apt-get update && apt-get install -y \
     bzip2 \
     libkrb5-dev \
+    libssl-dev \
     wget \
+    git \
```

Comment:
I don't think we need these right? Since we ended up copying the binary into `bin/bedToBigBed` in this repo